### ISSUE TITLE
Missing Metals under the new Ores System

### ISSUE THREAD

**Issue by Vaygrim** [Jozufozu/ExNihiloOmnia#131]
Previous versions of Ex Nihilo Omnia added its own metals (for Copper, Tin, Silver, Lead, Platinum, etc). This means that resources sieved and smelted would quite often come out as Ex Nihilo Omnia 'owned' ingots.

Now under your new system .. those ingots are totally gone. We had silver activated in your mod so we could have folks use it for Tinkers' weapons, and other mods would autodetect the presence of silver and use it in recipes. If we update to the 'new' E.N.O. we risk having all of our ingots 'non-vanilla ingots' vanish completely, as I have to designate an ingot from another mod in the JSON's. E.N.O. has no direct ingot ownership anymore.

Help? 

Because otherwise .. I literally can't roll this mod out in my pre-existing pack and I'll be 'version locked' using the edition BEFORE the forestry crashes.

**Comment by Vaygrim** [comment]
Ported test world after configuring the ores as best we could:
![2016-12-19_18 01 15](https://cloud.githubusercontent.com/assets/4295188/21333439/57d8bf24-c615-11e6-9105-2881be3a35bf.png)

**Comment by Jozufozu** [comment]
Can you tell me specifically what ingots are missing? I might be able to help then.

**Comment by RedeyeTheGoblin** [comment]
Silver, Copper and Tin are all missing from our upgrade upto v1.2.4. We can get around Copper and Tin, but ENO was the only source of Silver we had.

**Comment by Jozufozu** [comment]
You can add silver, copper and tin to the whitelist.txt file in "config/ExNihiloOmnia/registries/ore" and it should work fine.

**Comment by Vaygrim** [comment]
We can NOT get around Copper & Tin. Those are the two items with missing textures across the bottom of that Storage Crate screen shot.

**Comment by Vaygrim** [comment]
> You can add silver, copper and tin to the whitelist.txt file in "config/ExNihiloOmnia/registries/ore" and it should work fine.

Can you tell us HOW to do that? Your documentation on this whole new system structure is a bit ... sparse.

**Comment by Jozufozu** [comment]
Yeah... sorry about that. I'll work on it soon.
Your ore registry folder probably looks something like this: http://i.imgur.com/Jo8UFce.png
So go into "whitelist.txt" and add three lines (without quotes): "silver", "copper", and "tin"
So it looks like http://pastebin.com/H933YYAm

**Comment by Vaygrim** [comment]
What about the Dust Blocks for Iron and Gold? Those got transformed into "tile.remap_dust.name (#4723)" (same item for both metal types).

**Comment by Jozufozu** [comment]
They will turn into the new blocks if you put them in your inventory or place them in the world. I'm sorry for the inconvenience, but it was either that or corrupt the world.

**Comment by Vaygrim** [comment]
So I'm going to need to figure out how to inform my userbase of mostly clueless newbies and rookies how to correct this problem when -they- encounter it?

Oh this will be fun.

**Comment by Jozufozu** [comment]
I can't apologize enough for the trouble. I'll see if I can figure out a way to make the transition happen easier.

**Comment by Vaygrim** [comment]
For the record:
I'm not *trying* to find a way to make your existence harder: I know that mod authors already have to deal with a lot of idiots on a daily basis. I try my best express my appreciation for their efforts, and report bugs like a normal person. 

That being said, "Ex Nihilo Omnia" has been the bane of my existence in one way or another for the better part of several weeks now (and all from stuff you already know about or have had reported. Witchwater, unbreakable barrels, etc etc). So I'm just kind of frustrated. I apologize if it comes across as "Wow you are DUMB." That's not intended.

**Comment by Jozufozu** [comment]
No, I know that I have not been fully there in developing this mod, so I understand your frustration. This is the first mod and even the first dive into programming I've ever done. I never expected it to blow up like it did and I did not expect it to take up as much time as it has. If I had known, I might have been more reluctant to start, but I'm glad I did. I'll see what I can do to make up for my lack of support over the past few weeks.
